### ISSUE TITLE
Intermittent test failure on bookmarks accessibility test

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/dpul#1259]
Here's a failing test run: https://app.circleci.com/pipelines/github/pulibrary/dpul/867/workflows/a1d21968-2f9d-4f35-b5e2-346da7458a10/jobs/4256

I got it to fail locally on the 2nd run. From the html screenshot, there simply wasn't anything in the search results list.
